pandas/tools/merge.py

```diff
@@ -1436,13 +1438,51 @@ def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
     copy : boolean, default True
         If False, do not copy data unnecessarily
 
-    Notes
-    -----
-    The keys, levels, and names arguments are all optional
-
     Returns
     -------
     concatenated : type of objects
+
+    Notes
+    -----
```

Comment:
if you are going to add examples, then this needs to be more complete, using examples from here:http://pandas.pydata.org/pandas-docs/stable/merging.html#concatenating-objects

certainly not all of these, but a quick selection (as well as having this link would be fine).